### ISSUE TITLE
NVDA fails to behave properly when opening MS DOC/DOCX password protected files in MS Word 2010.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2827]
**Reported by Harich on 2012-11-26 22:56**
I've MS DOC and DOCX files protected with passwords. When I open them in MS Word 2010, I encounter following issues. 
1. After I hit enter on a password protected file to open it, the NVDA fails to speak the prompt to enter the password. 
2. I also hear NVDA startup audio feed, as if I had just restarted the NVDA. 
I've no idea if the above issue is also true for other versions of MS Word; I'm only using MS Word 2010.

**Comment by ehollig** [comment]
When I open a password protected Word documents, NVDA does read the prompt to enter the password. NVDA does not read the characters typed or read when I delete them, but that is a different issue stated by @michaelDCurran in #2716. As we are unable to contact the original poster in regards to the crash to debug the issue, closing as duplicate of #2716
